html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+
+    <div>
+      <label for="email">Email</label>
+    </div>
+
+    <input type="text" name="email" />
+
+    <div>
+      Why is coding cool?
+    </div>
+
+    <select name="Select an Option">
+      <option value="select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+      <option value="Because I can code a girlfriend">Because I can code a girlfriend</option>
+    </select>
+
+    <p>
+      Favorite Language (choose one)
+    </p>
+
+    <input type="radio" name="language" value="HTML" /> HTML
+    <input type="radio" name="language" value="CSS" /> CSS
+    <input type="radio" name="language" value="JavaScript" /> JavaScript
```

Comment:
These 3 `<input>` for the various languages each need their own unique `<label>`